src/ucp/tag/tag_send.c

```diff
@@ -233,8 +237,10 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_sync_nb,
 {
     ucp_request_param_t param = {
         .op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE | UCP_OP_ATTR_FIELD_CALLBACK,
+                        UCP_OP_ATTR_FIELD_MEMORY_TYPE,
         .cb.send      = (ucp_send_nbx_callback_t)cb,
-        .datatype     = datatype
+        .datatype     = datatype,
+        .memory_type  = UCS_MEMORY_TYPE_UNKNOWN
```

Comment:
no need to add UCP_OP_ATTR_FIELD_MEMORY_TYPE param, the default is already UNKNOWN